packages/math/src/pool/stable.ts

```diff
@@ -175,13 +249,14 @@ export function binarySearch(
     curOutput = makeOutput(curEstimate);
   }
 
-  return new Dec(0);
+  throw Error('binary search did not converge');
 }
 
-function compare_checkMultErrorTolerance(
+export function compare_checkMultErrorTolerance(
   expected: Dec,
   actual: Dec,
-  tolerance: Dec
+  tolerance: Dec,
+  roundingMode: string,
```

Comment:
A somewhat unique feature of TypeScript- we can enforce the shape of this string by replacing the `string` type with a mores specific union of literals: `"roundDown" | "roundUp"`.

Reply:
Neat!